wdqs-frontend/latest/entrypoint.sh

```diff
@@ -15,7 +15,7 @@ fi
 set -eu
 
 export DOLLAR='$'
-envsubst < /templates/custom-config.json > /usr/share/nginx/html/wikibase/custom-config.json
+envsubst < /templates/custom-config.json > /usr/share/nginx/html/default-config.json
```

Comment:
This needs to be custom-config, but not in the wikibase subfolder

Reply:
See current update, is that how it is supposed to be?

Reply:
> See current update, is that how it is supposed to be?

You still should avoid naming it "default-config.json" because it's not default. Only moving it out of the directory should be enough.
Now it comes the question of if we read this file. I should double check that.

Reply:
Yes has been changed see comment below.